products/compute/api.yaml

```diff
@@ -2416,6 +2416,21 @@ objects:
           lowercase letter, and all following characters must be a dash, lowercase
           letter, or digit, except the last character, which cannot be a dash.
         required: true
+      - !ruby/object:Api::Type::Array
+        name: candidateSubnets
+        description: |
+          Up to 16 candidate prefixes that can be used to restrict the allocation
+          of cloudRouterIpAddress and customerRouterIpAddress for this attachment.
+          All prefixes must be within link-local address space (169.254.0.0/16)
+          and must be /29 or shorter (/28, /27, etc). Google will attempt to select
+          an unused /29 from the supplied candidate prefix(es). The request will
+          fail if all possible /29s are in use on Google's edge. If not supplied,
+          Google will randomly select an unused /29 from all of link-local space.
+        item_type: Api::Type::String
+      - !ruby/object:Api::Type::Integer
+        name: vlanTag8021q
```

Comment:
The upstream docs for this field state that it's

> Only specified at creation time.

Which means that part of this resource is updatable; I know it's more of a historical question, but do you know why we've marked the entire resource as `input`?

Reply:
Yep - there's a patch method that we aren't supporting because we want to keep this resource as simple as possible (due to lack of testability).